query.css

```diff
@@ -0,0 +1,356 @@
+@media only screen and (min-width: 768px) {
+  .container {
+    height: 100%;
+    width: 100%;
+  }
+
+  #fixed {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    background-color: white;
+  }
+
+  .mobile_menu_icon {
+    display: none;
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 50px;
+  }
+
+  .logo {
+    width: 150px;
+    height: 30px;
+    font-size: 20px;
+  }
+
+  .social a {
+    padding: 15px 25px 15px 10px;
+  }
+
+  .desktop-nav {
+    display: flex;
+    float: left;
+    align-items: center;
+  }
+
+  .desktop-nav ul li {
+    display: inline-flex;
+    padding: 0 20px;
+  }
+
+  .desktop-nav ul li a {
+    color: #344563;
+  }
+
+  .hero {
+    background-image: url(./assets/desktop/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    justify-content: center;
+    align-items: center;
+    border-bottom-left-radius: 100px;
+    height: 750px;
+  }
+
+  .author {
+    margin-left: 200px;
+    padding-top: 200px;
+    width: 30%;
+    justify-content: center;
+    letter-spacing: 0.37px;
+  }
+
+  .bio {
+    margin-left: 200px;
+    width: 60%;
+    height: auto;
+    letter-spacing: 0.37px;
+  }
+
+  .connect_button {
+    margin-left: 200px;
+    font-size: 20px;
+  }
+
+  .social ol {
+    margin-left: 200px;
+  }
+
+  .desktop-card,
```

Comment:
:red_circle: Kindly notice the white space after the project description on odd cards and before on even cards. Kindly reduce the white space to match the Figma design

Your Design
![yrextraspace](https://user-images.githubusercontent.com/26736582/200090122-d71d33de-8c7c-43b4-a68d-5d2fb8f09dde.png)

Figma Design

![extraspace](https://user-images.githubusercontent.com/26736582/200090143-7b49a5a5-8b0f-4ea8-83e8-c4edc81da9ce.png)